include/deal.II/grid/grid_tools.h

```diff
@@ -797,13 +797,17 @@ namespace GridTools
     const std::vector<std::vector<BoundingBox<spacedim>>> &global_bboxes);
 
   /**
-   * Return a map of index:Point<spacedim>, containing the used vertices of the
-   * given `container`. The key of the returned map is the global index in the
-   * triangulation. The used vertices are obtained by looping over all cells,
+   * Return a map `vertex index -> Point<spacedim>`, containing the used
+   * vertices of the given `container`. The key of the returned map (i.e.,
+   * the first element of the pair above) is the global index in the
+   * triangulation, whereas the value of each pair is the physical
+   * location of the corresponding vertex. The used vertices are obtained by
+   * looping over all cells,
    * and querying for each cell where its vertices are through the (optional)
    * `mapping` argument.
    *
-   * In serial and parallel::shared triangulations the size of the returned map
+   * In serial Triangulation objects and parallel::shared::Triangulations
+   * objects, the size of the returned map
    * equals Triangulation::n_used_vertices() (not Triangulation::n_vertices()).
    * Note that in parallel::distributed triangulations only vertices in locally
```

Comment:
Would you mind fixing the doxygen link here while you are updating the documentation of this function?